protostar/commands/invoke_command.py

```diff
@@ -44,7 +44,7 @@ def arguments(self):
             ProtostarArgument(
```

Comment:
I think "account-address" arg in the `SignableCommandUtil` should also be updated.

Reply:
done